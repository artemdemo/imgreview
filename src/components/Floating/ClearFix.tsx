import styled from 'styled-components';

const ClearFix = styled.div`
  &::after {
    content: '';
    display: table;
    clear: both;
  }
`;

export default ClearFix;
