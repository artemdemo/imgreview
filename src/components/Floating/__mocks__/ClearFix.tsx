import React from 'react';

const ClearFix = (props) => <div data-mock="ClearFix">{props.children}</div>;

export default ClearFix;
