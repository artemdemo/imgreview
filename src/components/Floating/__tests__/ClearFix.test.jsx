import React from 'react';
import renderer from 'react-test-renderer';
import ClearFix from '../ClearFix';

describe('ClearFix', () => {
  it('render empty', () => {
    const tree = renderer.create(<ClearFix />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
