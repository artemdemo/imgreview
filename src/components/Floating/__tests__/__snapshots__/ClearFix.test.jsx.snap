// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClearFix render empty 1`] = `
<div
  className="sc-bdVaJa flWwzy"
/>
`;
